### PR TITLE
Create ru_ru.json

### PR DESCRIPTION
**Pull request by Smollet777**

### CODE REVIEW

File: src/generated/resources/assets/mininggadgets/lang/ru_ru.json [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  "mininggadgets.tooltip.screen.shrink": "Блоки сжимаются",
+  "mininggadgets.tooltip.screen.size": "Размер: %1$d x %1$d",
+  "mininggadgets.tooltip.screen.toggle_upgrades": "Переключить улучшения",
+  "mininggadgets.tooltip.screen.visual_settings": "Визуальные настройки",
+  "mininggadgets.tooltip.screen.visuals_menu": "Редактировать визуальные эффекты",
+  "mininggadgets.tooltip.screen.volume": "Звук",
+  "mininggadgets.tooltip.screen.whitelist": "Белый список активен",
+  "mininggadgets.tooltip.single.filters": "Фильтры",
+  "mininggadgets.tooltip.single.insert": "Вставить",
+  "mininggadgets.upgrade.replacement": "Улучшение: "
```
Comment (MichaelHillcox) [inline src/generated/resources/assets/mininggadgets/lang/ru_ru.json]:
```suggestion
  "mininggadgets.upgrade.replacement": "Улучшение: ",
```

**Comment by MichaelHillcox** [comment]
If you can apply my suggestion and move the file out of generated and place it in `src/main/resources/assets/mininggadgets/lang/ru_ru.json` then I'll accept the pr :+1: thanks for the hard work ❤️

**Comment by Smollet777** [comment]
Oops! My bad. Thanks

**Comment by MichaelHillcox** [comment]
Thanks for the translations :+1: :heart:

**Pull request merged**